lib/utils/OpenFoodAPIConfiguration.dart

```diff
@@ -29,6 +29,14 @@ class OpenFoodAPIConfiguration {
   ///Uri host of the test requests to the backend
   static String uriTestHost = 'world.openfoodfacts.net';
 
+  ///Url base for images in prod: needs to match the domain of uriProdHost
+  static String ImageProdUrlBase =
```

Comment:
It would be better here to add the `const` keyword.
The same goes for line 37

Reply:
The other strings in this file don't have const, I'm guessing it's because we want to be able to update them at runtime. (e.g. in smoothie we can set any API server url through the advanced settings mode)

Reply:
Mmmm good point, that's not how I manage this kind of config in my other projects, but why not.
Forget this comment in that case